CMakeLists.txt

```diff
@@ -102,7 +102,7 @@ find_package(Boost COMPONENTS filesystem)
 include_directories(${Boost_INCLUDE_DIRS})
 
 # TODO: utils-urdf should only be required for aikido::perception.
-find_package(DART REQUIRED COMPONENTS optimizer-nlopt utils-urdf CONFIG)
+find_package(DART 6.1.0 REQUIRED COMPONENTS optimizer-nlopt utils-urdf CONFIG)
```

Comment:
I think this will break with *any* release other than DART 6.1.0. Since we only break backwards compatibility on major releases, I think we're better off `find_package(DART ...)` and then checking that `DART_VERSION` is greater than or equal to `6.1.0` and less than `7.0.0`. Thoughts?